From: Net-SNMP Packaging Team <pkg-net-snmp-devel@lists.alioth.debian.org>
Date: Thu, 18 Jun 2015 06:12:04 +0900
Subject: _searchdirs

Makefile patch to include libwrap and libsensors libraries and install extra
MIB files.
---
 local/mib2c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/local/mib2c b/local/mib2c
index 2016f06..6926321 100755
--- a/local/mib2c
+++ b/local/mib2c
@@ -61,8 +61,9 @@ $currentlevel = -1;
 if($ENV{MIB2C_DIR}) {
    push @def_search_dirs, split(/:/, $ENV{MIB2C_DIR});
 }
-push @def_search_dirs, "/usr/local/share/snmp/";
-push @def_search_dirs, "/usr/local/share/snmp/mib2c-data";
+push @def_search_dirs, "/etc/snmp/";
+push @def_search_dirs, "/usr/share/snmp/";
+push @def_search_dirs, "/usr/share/snmp/mib2c-data";
 push @def_search_dirs, "./mib2c-conf.d";
 
 sub usage {
